src/components/modals/modification/ConvertType.vue

```diff
@@ -0,0 +1,144 @@
+<script setup lang="ts">
+import type Scale from "@/scale";
+import { onMounted, ref } from "vue";
+import Modal from "@/components/ModalDialog.vue";
+import type { IntervalOptions, IntervalType } from "@/interval";
+import { Fraction } from "xen-dev-utils";
+
+const props = defineProps<{
+  scale: Scale;
+  centsFractionDigits: number;
+  decimalFractionDigits: number;
+}>();
+
+const emit = defineEmits(["update:scale", "cancel"]);
+const type = ref<IntervalType>("cents");
+const centsFractionDigits = ref(3);
+const decimalFractionDigits = ref(5);
+const preferredNumerator = ref<number>(0);
+const preferredDenominator = ref<number>(0);
+const preferredEtDenominator = ref<number>(0);
+const preferredEtEquaveNumerator = ref<number>(0);
+const preferredEtEquaveDenominator = ref<number>(1);
+
+onMounted(() => {
+  centsFractionDigits.value = props.centsFractionDigits;
+  decimalFractionDigits.value = props.decimalFractionDigits;
+});
+
+function modify() {
+  const options: IntervalOptions = {
+    centsFractionDigits: centsFractionDigits.value,
+    decimalFractionDigits: decimalFractionDigits.value,
+  };
+  if (preferredNumerator.value) {
+    options.preferredNumerator = preferredNumerator.value;
+  }
+  if (preferredDenominator.value) {
+    options.preferredDenominator = preferredDenominator.value;
+  }
+  if (preferredEtDenominator.value) {
+    options.preferredEtDenominator = preferredEtDenominator.value;
+  }
+  if (preferredEtEquaveNumerator.value) {
+    options.preferredEtEquave = new Fraction(
+      preferredEtEquaveNumerator.value,
+      preferredEtEquaveDenominator.value
+    );
+  }
+
+  emit("update:scale", props.scale.asType(type.value, options));
+}
+</script>
+
+<template>
+  <Modal @confirm="modify" @cancel="$emit('cancel')">
+    <template #header>
+      <h2>Convert line types</h2>
```

Comment:
This is a useful new feature and works well on my local!

It might be the first time we have exposed users to the idea of "line types" as far as I remember. Are any these alternative phrases easier for average users to grasp? Convert interval types, convert interval notation, convert scale notation... Any other suggestions?

Reply:
The closest I've seen SW1 coming to naming these in user space is *value* in the User Guide.
> Values with a . are cents values: e.g. 701.955
> Values with slash (/) are ratios: e.g. 3/2
> Values with a backslash (n\m) are n degrees of m-EDO: e.g. 7\12
> Values with comma (,) are decimals: e.g. 1,5

So maybe *convert interval values*?